src/i18n/en.json

```diff
@@ -41,26 +41,35 @@
   "processFourTitle": "Realistic deliveries",
   "processFourText": "We know how frustrating it's to have a project delayed and be the last person to know. Therefore, the contact with our customers is continuous. We set realistic dates for our deliveries and keep a channel always opened to explain the evolution of the work.",
   "processFiveTitle": "Total freedom",
-  "processFiveText": "We create projects with clean, functional and modular code. So if at some point our customers wants to internalize improvements or add another professional for a new functionality, there will be no problems. The quality, ideas and snappiness are what keep our customers around and not because you are trapped by our code.",
+  "processFiveText": "We create projects with clean, functional and modular code. So if at some point our customers wants to internalize improvements or add another professional for a new functionality, there will be no problems. The quality, ideas and snappiness are what keep our customers around instead of them being trapped by our code",
```

Comment:
```suggestion
  "processFiveText": "We create projects with clean, functional and modular code. So if at some point our customers want to internalize improvements or add another professional for new functionality, there will be no problems. The quality, ideas and snappiness are what keep our customers around instead of them being trapped by our code.",
```